cmd/deployment/show.go

```diff
@@ -90,4 +111,5 @@ func initShowFlags() {
 	showCmd.Flags().Bool("plan-history", false, "Shows the deployment plan history")
 	showCmd.Flags().BoolP("metadata", "m", false, "Shows the deployment metadata")
 	showCmd.Flags().BoolP("settings", "s", false, "Shows the deployment settings")
+	showCmd.Flags().Bool("generate-payload", false, "Outputs a JSON formatted payload which can be used as an argument for the --file flag on update command.")
```

Comment:
```suggestion
	showCmd.Flags().Bool("generate-payload", false, "Outputs JSON which can be used as an argument for the --file flag with the update command.")
```

(If this is technically accurate, can you simplify the wording as suggested?)